cockroachcloud/private-clusters.md

```diff
@@ -0,0 +1,46 @@
+---
```

Comment:
Noting that this topic actually comes from https://github.com/cockroachdb/docs/pull/15342